src/main/java/frc/robot/commands/drivingCommands/DriveAndLoadingAimCommand.java

```diff
@@ -29,10 +31,12 @@ public void initialize() {
     @Override
     public void execute() {
         if (limelight.hasValidTarget()) {
+            slowSpeed = limelight.getTargetArea()/100;
```

Comment:
As we get closer and lose sight of the target, the target area percentage of the image will get smaller which, in turn will speed up the robot. Can you ensure that `slowSpeed` is only set if it makes the robot go slower?

Reply:
It only sets the slowspeed when it sees the target , when it gets too close it will default to it's last seen value. I could see it not staying full in frame, but when that happens we typically loose the target.